### PR TITLE
Update ws-security saml.1 fat project to run with jakarta ee9 features

### PR DESCRIPTION
**Pull request by meiaus**
update com.ibm.ws.wssecurity_fat.wsscxf.saml.1 with jakarta ee9

### CODE REVIEW

**Comment by meiaus** [comment]
#run-libby-bot

**Comment by meiaus** [comment]
#build

**Review by jimmy1wu — APPROVED** [review]

File: dev/com.ibm.ws.wssecurity_fat.wsscxf.saml.1/bnd.bnd [hunk 1]
```diff
@@ -21,11 +21,24 @@ src: \
...
+  jsp-2.3, \
+  pages-3.0, \
+  servlet-4.0, \
+  servlet-5.0, \
+  appSecurity-3.0, \
+  appSecurity-4.0, \
+  cdi-2.0, \
+  cdi-3.0, \
+  el-3.0, \
+  expressionlanguage-4.0
```
Comment (jimmy1wu) [inline dev/com.ibm.ws.wssecurity_fat.wsscxf.saml.1/bnd.bnd:39]: was federatedregistry-1.0 and ldapregistry-3.0 purposefully not included in this new list?

Comment (meiaus) [reply dev/com.ibm.ws.wssecurity_fat.wsscxf.saml.1/bnd.bnd:39]: yes. they are not used in these FAT (ldap user registry is not used but basicRegistry). They were used at one point during FAT migration and after learning more about the testing, it's time for minor cleanup.

**Comment by arunavemulapalli** [comment]
for #12449

**Pull request merged**